SU-SWS/stanford_paragraph_types
# EARTH-163:

# READY FOR REVIEW

# Summary
- Sets the WYSIWYG paragraph to be centered using the basic layout by default.

# Needed By (Date)
- Sometime this sprint

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Import feature settings `drush features-import-all`
3. Clear cache
4. Create wysiwyg paragraph and see it centered on the page.

# Affected Projects or Products
- EARTH-163

# Associated Issues and/or People
- None

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
